.github/workflows/docker-image.yaml

```
@@ -0,0 +1,50 @@
+name: Docker Image Update
+
+on:
+  release:
+    types: [published]
+  pull_request:
+    paths:
+      - Dockerfile
+      - scripts/dev_cli.sh
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-20.04
```
`ubuntu-latest` may point to 22.04 in the near future. So let's use `ubuntu-20.04` explicitly until we check the portability.